data-esnext.js

```diff
@@ -271,9 +284,10 @@ exports.tests = [
     },
     {
       name: 'SIMD.%booleanType%.and',
+      spec: 'https://tc39.github.io/ecmascript_simd/#simd-and',
       mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SIMD/and',
       exec: function(){/*
-        return typeof SIMD.Bool32x4.and === 'function' && typeof SIMD.Bool16x8.and === 'function' && typeof SIMD.Bool8x16.and === 'function';
+        return typeof SIMD.Bool32x4.and === 'function' && typeof SIMD.Int32x4.and === 'function' && typeof SIMD.Uint32x4.and === 'function';
```

Comment:
Because the spec says this operation is available for all these types:

SIMD.Int8x16.and(a, b)
SIMD.Int16x8.and(a, b)
SIMD.Int32x4.and(a, b)

SIMD.Uint8x16.and(a, b)
SIMD.Uint16x8.and(a, b) 
SIMD.Uint32x4.and(a, b)

SIMD.Bool8x16.and(a, b) 
SIMD.Bool16x8.and(a, b) 
SIMD.Bool32x4.and(a, b)

I felt it was better to check all types than all sizes of one type.